scalyr_agent/log_processing.py

```diff
@@ -2906,6 +2923,7 @@ def __rename_log_file(self, matched_file, log_config):
                     except Exception as e:
                         log.warn(
                             "Error matching regular expression '%s' and replacing with '%s'.  %s"
+                            # 2->TODO same as 2046 line.
```

Comment:
Line numbers can change... repeat the comment or use some more stable way to refer to that first comment.

Reply:
Done